common/script_values/imf_values.txt

```diff
@@ -1 +1,818 @@
-﻿
\ No newline at end of file
+﻿global_gdp = {
```

Comment:
Suggest rename, current name produces error on startup and will overshadow the event_target of the same name